docs/rum_collection/crash_reporting.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Crash Reporting (beta)
+kind: documentation
+further_reading:
+  - link: "https://github.com/DataDog/dd-sdk-ios"
+    tag: "Github"
+    text: "dd-sdk-ios Source code"
+  - link: "/real_user_monitoring"
+    tag: "Documentation"
+    text: "Datadog Real User Monitoring"
+---
+
+Enable iOS crash reporting and error tracking to get comprehensive crash reports, and error trends in RUM UI. With this beta feature, you get access to 
+
+ - Aggregated iOS crash data and RUM crash attributes
+ - Desymbolicated iOS error reports
+ - Trend analysis with iOS error tracking
+
+This guide follows the setup for iOS crash reporting and error tracking in the following steps
+ - Add crash reporting
+ - Desymbolicate error reports
+ - Verify setup
+
+### Add crash reporting 
+
+If you have not set up the SDK yet, follow the [in-app setup instructions][1] or refer to the [iOS RUM setup documentation][2]. Upgrade to [dd-sdk-ios v1.7.0+][3] to get access to iOS crash reporting and error tracking. Update your initialization snippet to include crash reporting:
+
+
+```
+Datadog.initialize(
+    appContext: .init(),
+    trackingConsent: .pending,
+    configuration: Datadog.Configuration
+    .builderUsing(
+        rumApplicationID: "APP_ID",
+        clientToken: "CLIENT_TOKEN",
+        environment: "ENVIRONMENT"
```

Comment:
To unify with [existing docs](https://docs.datadoghq.com/real_user_monitoring/ios/?tab=us#setup):
```suggestion
        rumApplicationID: "<rum_application_id>",
        clientToken: "<client_token>",
        environment: "<environment_name>"
```